play-services-nearby-core-ui/src/main/res/xml/filepaths.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths>
+    <files-path path="." name="files" />
```

Comment:
Exposing everything in the /files folder is bad practice. Instead, please only expose a specific folder for these exports.